js/interactions.md

```diff
@@ -142,8 +142,28 @@ When using React, you can use *ChatBot* with following properties;
     welcomeMessage="Welcome, how can I help you today?"
     onComplete={this.handleComplete.bind(this)}
     clearOnComplete={true}
+    conversations={false}
```

Comment:
`conversations` prop it sounds a little odd. Maybe `conversationEnabled` or `conversationModeOn`?

Reply:
good point, `conversationModeOn` is much more descriptive